src/main/java/io/github/sefiraat/crystamaehistoria/utils/StoryUtils.java

```diff
@@ -95,7 +96,7 @@ public static boolean isStoried(ItemStack itemStack) {
     /**
      * Sets the ItemStack's PDC Storied to True. Also sets an initial story object
      *
-     * @param itemStack The {@link ItemStack} whos meta will have the PDC element added to
+     * @param itemStack The {@link ItemStack} whose meta will have the PDC element added to
```

Comment:
Man, I swear I got A* in English in school - I don't deserve to be trusted on peoples servers xD